cleanup/user-cluster.yml

```diff
@@ -92,4 +92,18 @@ rules:
   verbs:
   - list
   - get
+  - delete
```

Comment:
Was there an error deleting the job without this permission before?

Reply:
delete is already there, it's just shifted to line number 109